### PR TITLE
Add exploding subheading under dropdown on retention policy picker

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers

### CODE REVIEW

File: shared/teams/team/settings-tab/retention/container.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ const mapStateToProps = (state, ownProps: OwnProps) => {
     throw new Error(`RetentionPicker: impossible entityType encountered: ${entityType}`)
   }
 
+  const policyIsExploding = policy.type === 'explode'
```
Comment (buoyad) [inline shared/teams/team/settings-tab/retention/container.js]: `|| (policy.type === 'inherit' && teamPolicy.type === 'explode')`

File: shared/teams/team/settings-tab/retention/index.js [hunk 2]
```diff
@@ -200,6 +201,17 @@ class _RetentionPicker extends React.Component<Kb.PropsWithOverlay<Props>, State
             Individual channels can override this.
           </Kb.Text>
         )}
+        {this.props.policyIsExploding && (
```
Comment (buoyad) [inline shared/teams/team/settings-tab/retention/index.js]: This is above 'Individual channels can override this.' in designs. Also the gap between them is too big in app
image
design:
image

**Comment by cjb** [comment]
@buoyad Thanks for catching those!  Ready for another look.

screen shot 2019-02-06 at 9 28 07 am

File: shared/teams/team/settings-tab/retention/index.js [hunk 3]
```diff
@@ -196,22 +196,22 @@ class _RetentionPicker extends React.Component<Kb.PropsWithOverlay<Props>, State
...
         {this.props.policyIsExploding && (
           <Kb.Box2 direction="horizontal" alignItems="center" fullWidth={true} gap="xtiny">
             <Kb.Text type="BodySmall">Participants will see their message explode.</Kb.Text>
-            <Kb.Icon
-              color={Styles.globalColors.black_75}
-              fontSize={Styles.isMobile ? 44 : 35}
-              style={boomIconStyle}
-              type="iconfont-boom"
-            />
+            <Kb.Box style={boomIconBoxStyle}>
```
Comment (buoyad) [inline shared/teams/team/settings-tab/retention/index.js]: This goes off the right side of the screen in the infopanel on on the iPhone 5s sim. Weirdly, the text wraps correctly on the team settings page.
image
image

Comment (cjb) [reply shared/teams/team/settings-tab/retention/index.js]: @buoyad Removing a `width: '100%'` got the wrap working, what do you think?

**Pull request merged**